.travis.yml

```diff
@@ -7,6 +7,10 @@ services:
   - redis-server
 
 script:
+  - rm -rf $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4/backends
+  - rm -rf $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4/ui
```

Comment:
Won't this fail? What if $GOPATH/src/gopkg.in/gilmour-libs/gilmour-e-go.v4/ doesn't exist, which in most likelihood might not be there.
Maybe you should create that directory path first?